content/tutorials/portenta-h7/por-ard-lvgl/content.md

```diff
@@ -54,18 +51,15 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
    
 4. Adding a Button widget 
 
-   Lets add custom button widget using the `lv_btn` object, This object allows you to have an inner label whose text can be easily configure. The following lines of code initialises the button variable and its label which you can add it at the beginning of the sketch. 
+   Lets add custom button widget using the `lv_btn` object, This object allows you to have an inner label whose text can be easily configure. The following lines of code initialises the button variable and its label (`myCustomLabel`) which you can add it at the beginning of the sketch. 
 
    ```cpp
    static lv_obj_t *lv_btn;
    static lv_obj_t *myCustomLabel;
    ```
-
-   ![por_ard_lvgl_create_variables](assets/por_ard_lvgl_create_variables.png)
-
    
 
-   Then at the end of the `setup()` you can configure the type, style and the position of the button. 
+   Then at the end of the `setup()` you can configure the type, style and the position of the button and label. 
```

Comment:
If you are combining 3 and 4 then this step can be called as "Configuring the button widget"